manim/mobject/svg/text_mobject.py

```diff
@@ -207,7 +207,10 @@ def __init__(
             height=height,
             width=width,
             unpack_groups=unpack_groups,
-            color=color ** config,
+            color=color,
+            fill_opacity=fill_opacity,
+            stroke_width=stroke_width,
+            should_center=should_center,
```

Comment:
I think (although I am not 100% sure; CC @cobordism @leotrs) that the constructor of `SVGMobject` here should receive all input arguments from lines 142 to 148 (https://github.com/ManimCommunity/manim/pull/858/files#diff-1cf583cd8792705997f6ee46cd7118aa3d3d52513aeeaab03805586982f02b25R142-R148) as these (should) specify properties of the final Mobject.

Could you please try to add the ones from there that are not passed yet, as well as a `**kwargs` for all other keyword args? (As far as I can see, they are not processed anywhere else in `CairoText` and thus should be propagated.)

Reply:
The arguments from lines 142 to 148 should all be there (line 206-213). Or do I miss something?

```**kwargs``` was indeed missing. I just added it

Reply:
Oh, sorry! My bad, I'm bad at counting. 😄